README.md

```diff
@@ -395,6 +424,59 @@ Or:
 $ HOROVOD_GPU_ALLREDUCE=NCCL HOROVOD_NCCL_INCLUDE=/path/to/nccl/include HOROVOD_NCCL_LIB=/path/to/nccl/lib pip install --no-cache-dir horovod
 ```
 
+### NCCL 2 is not found during runtime
```

Comment:
Not for this review but eventually we might want to move all this to a separate docs folder & have it linked like how it is done in https://github.com/uber/pyflame (check FAQ link).